concent_api/core/views.py

```diff
@@ -554,6 +630,16 @@ def handle_send_force_get_task_result(request, client_message: message.concents.
             requestor_public_key = client_public_key,
             status               = ReceiveStatus,
         )
+        store_or_update_subtask(
```

Comment:
@cameel Please check if `next_deadline` is defined correctly and if correct related messages are stored.